obfx_modules/mystock-import/js/src/components/index.js

```diff
@@ -0,0 +1,20 @@
+import PhotoList from "./PhotoList";
+import SetFeaturedImage from "./SetFeaturedImage";
+import InsertImage from "./InsertImage";
+
+const { PluginSidebar, PluginSidebarMoreMenuItem } = wp.editPost;
+
+const Mystock = () => {
+	return (
+		<PluginSidebar
+			icon="camera"
+			name="mystock-sidebar"
+			title="MyStockPhotos"
```

Comment:
You need to use `const { __ } = wp.i18n` package for translations.

Reply:
@HardeepAsrani There are no strings that need to be translated here and all the strings are localized. Should I remove the localized strings and use const { __ } = wp.i18n ?

Reply:
If they're localized then sure. But then you should also use that for the name of the plugin as MyStockPhotos will remian same but in translated-alphabets.